Testing/functional/e2e/cypress/integration/services/webClient/comment.js

```diff
@@ -35,24 +35,22 @@ describe('WebClient Comment Service', () => {
     }) 
 
     it('Verify Get Comment Authorized', () => {
-        cy.fixture('WebClientService/comments.json').then((expectedResponse) => {
```

Comment:
this is not validating the content of the comments anymore

Reply:
No I don't think we want to make this test depending on a specific list of comments right? Currently, there is a comment in the dev_db which fails this test.
Unless we want to insert comments in these tests then validate the list.
@FuriousLlama @sslaws @tiagograf-AOT any thoughs?

Reply:
per our team discussion today, I will add a script to the seed.sql to cleanup comments from db and revert code back to the ogirinal one.